share/wake/lib/system/http.wake

```diff
@@ -207,7 +216,9 @@ def makeCurlCmd ((HttpRequest url method headers body formData): HttpRequest) (e
 
 # Makes the request specified by request and parses into a HttpResponse.
 #
-# The default method to make a request. It should be used when the response is string encodable.
+# The standard method to make a request. When the response is string encodabled it must be used.
```

Comment:
`encodeabled` -> `encodeable`.  Also, the reversal of the clauses in the sentence make it harder to figure out "it" means "this function".